README.md

```diff
@@ -3,7 +3,10 @@
 A Chrome extension that brings the simplicity of Google Inbox to Gmail.
 
 ## How to use Simplify
-Add the extension and refresh Gmail: <https://chrome.google.com/webstore/detail/simplify-gmail/pbmlfaiicoikhdbjagjbglnbfcbcojpj>
+Add the extension and refresh Gmail:
+
+- [Chrome](https://chrome.google.com/webstore/detail/simplify-gmail/pbmlfaiicoikhdbjagjbglnbfcbcojpj)
+- [Firefox](https://addons.mozilla.org/pt-PT/firefox/addon/simplifygmail/)
```

Comment:
I guess it's better to use en-US here instead of pt-PT as most people would at least understand EN 🙄

Reply:
You're right! These auto localization URLs are tricky!

Reply:
https://addons.mozilla.org/firefox/addon/simplifygmail/ will auto detect your local.